resources/public/api/conf/1.0/retrieve_se_bsas.raml

```diff
@@ -0,0 +1,26 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatBsasId
+  - errors.formatAdjustedStatus
+  - errors.ruleNotSelfEmployment
+  - errors.ruleNoAdjustmentsMade
+  - errors.clientOrAgentNotAuthorised
+  - errors.notFound
+  - queryParameters.adjustedStatusOptional
```

Comment:
This comes up as type `string` in the documentation, but it should be a Boolean.

Reply:
Updated.